contentcuration/contentcuration/view/internal_views.py

```diff
@@ -257,3 +311,8 @@ def create_exercises(node, data):
             order += 1
             question_obj.save()
             map_files_to_assessment_item(question_obj, question['files'])
+
+def delete_tree(node):
+    for child in node.children.all():
+        delete_tree(child)
+    node.delete()
```

Comment:
The default behavior, if you just say `node.delete()`, is for all models pointing onto here to be "cascade deleted".

https://docs.djangoproject.com/en/1.10/ref/models/fields/#foreignkey